### PR TITLE
Bump draft links to -05, and use the new directives

### PR DESCRIPTION
**Pull request by joker314**
All the links were pointing to the -04 draft. However, the -05 draft has been [published
](https://tools.ietf.org/html/draft-foudil-securitytxt-05)

Hence, update all those links. I used a regular expression matching for "-04" to do this.

In addition, make the following changes:
- Change the form for generating security.txt files so that it lists the new version of the directives
- Use more succint notation in the JavaScript
- Expand the camelCase -> directive converter to be able to handle multiple words
- Use the `var` keyword to ensure variables stay in their scope only

### CODE REVIEW

**Comment by joker314** [comment]
Oh and we need to add/remove some of the directives!

**Comment by joker314** [comment]
Okay, I've updated all the directives and the links, and also changed the JavaScript a little. It should all still work in IE9+ (as before), but I haven't tested.

I think we should add some validation (e.g., make Contact a required field, and explain to the user if they fail to complete it). I'll file an issue for it.

**Comment by joker314** [comment]
Thanks @EdOverflow, that's super cool, so I've updated to the special links so that we'll have less work in future.

File: index.html [hunk 1]
```diff
@@ -30,43 +30,43 @@ <h1 class="title">Step 1</h1>
...
-                    <input name="permission" id="permission" class="input" placeholder="none">
+                    <input name="preferredLanguages" id="preferredLanguages" class="input" placeholder="en, es, ru">
                 </div>
             </div>
             <div class="field">
-                <label for="policy">Policy: <a target="_blank" rel="noopener" href="https://tools.ietf.org/html/draft-foudil-securitytxt-04#section-3.4.6">(description)</a></label>
+                <label for="canonical">Canonical: <a target="_blank" rel="noopener" href="https://tools.ietf.org/html/draft-foudil-securitytxt#section-3.5.2">(description)</a></label>
                 <div class="control">
-                    <input name="policy" id="policy" class="input" placeholder="https://example.com/security-policy.html">
+                    <input name="canonical" id="canonical" class="input" placeholder="https://example.com/.well-known/security.txt.sig">
```
Comment (nightwatchcyber) [inline index.html]: This should say "https://example.com/.well-known/security.txt" without the ".sig" in the end

Comment (joker314) [reply index.html]: Thanks for catching that @nightwatchcyber! I've gone ahead and fixed it.

**Pull request merged**